manim/mobject/svg/text_mobject.py

```diff
@@ -288,12 +309,13 @@ def text2svg(self):
         size = self.size * 10
         lsh = self.lsh * 10
 
-        if self.font == '':
-            if NOT_SETTING_FONT_MSG != '':
+        if self.font == "":
+            if NOT_SETTING_FONT_MSG != "":
                 logger.warning(NOT_SETTING_FONT_MSG)
-        dir_name = file_writer_config['text_dir']
+
+        dir_name = config["text_dir"]
```

Comment:
```suggestion
        dir_name = file_writer_config["text_dir"]
```

Same here. I think `file_writer_config` needs to be imported again as well...